src/test/java/com/example/eventapplication/user/UserServiceImplTest.java

```diff
@@ -0,0 +1,170 @@
+package com.example.eventapplication.user;
+
+import com.example.eventapplication.notification.dto.NotificationDto;
+import com.example.eventapplication.notification.entity.NotificationEntity;
+import com.example.eventapplication.user.dto.UserDto;
+import com.example.eventapplication.user.entity.UserEntity;
+import com.example.eventapplication.user.exception.UserServiceException;
+import com.example.eventapplication.user.repository.UserRepository;
+import com.example.eventapplication.user.service.UserReadService;
+import com.example.eventapplication.user.service.UserService;
+import com.example.eventapplication.user.service.UserUpdateService;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SpringBootTest
+public class UserServiceImplTest {
+
+    @Autowired
+    private UserService userService;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserReadService userReadService;
+
+    @Autowired
+    private UserUpdateService userUpdateService;
+
+    @AfterEach
+    void tearDown() {
+        userRepository.deleteAll();
+    }
+
+    @Test
+    public void should_create_new_user_successfully() {
+        //given
+        UserDto exampleDto = new UserDto(
+                "Jason",
+                "Warsaw",
+                "jason@gmail.com");
+
+        UserEntity referenceEntity = new UserEntity(
```

Comment:
wiem, że wygodnie, ale tak się nie robi. Jeżeli dojdzie logika do encji (który w tym projkecie jest też obiektem domenowym, albo możliwość zmiany stanu to ten test też się wywali i to nie dlatego że user się zmienił, tylko to referenceEntity przestanie działać. 

Tu trzeba normalnie pole po polu porównać